src/components/Sum.jsx

```diff
@@ -1 +1,12 @@
-// export function Sum() {}
+import React from 'react';
+
+export function Sum({ a, b }) {
+  return (
+    <p> Sum of {a} and {b} is {a + b}</p>
```

Comment:
```suggestion
    <p>Sum of {a} and {b} is {a + b}</p>
```
Fix it to pass the tests